packages/commerce-sdk-react/src/hooks/useDNT.ts

```diff
@@ -8,27 +8,34 @@ import useAuthContext from './useAuthContext'
 
 interface useDntReturn {
     dntStatus: boolean | undefined
+    finalDntValue: boolean | undefined
     updateDNT: (preference: boolean | null) => Promise<void>
 }
 
 /**
  * Hook that returns
- * dntStatus - a boolean indicating the current DNT preference
- * updateDNT - a function that takes a DNT preference and creates the dw_dnt
- *              cookie and reauthroizes with SLAS
- *
+ * dntStatus - a boolean indicating the DNT user choice status. Used to determine
+ *              if the consent tracking form should be rendered
+ * finalDntValue - a boolean indicating the current DNT preference to apply to
+ *              analytics layers. Takes defaultDnt into account when dntStatus is undefined.
+ *              If defaultDnt is undefined as well, then PWA Kit default is used.
```

Comment:
Nit: Let's avoid having the name `pwa-kit` as technically it is currently used only for pwa-kit, but it could be used for something other than pwa-kit. Let's replace it with `the app` or `storefront`

Reply:
Since `defaultDnt` is defined in the template retail react app, I feel like it could be confusing if we call the default in the SDK `app default or storefront default`. Would SDK default be a good choice?